_data/education.yml

```diff
@@ -43,6 +43,15 @@ websites:
       exceptions:
           text: "SMS-capable phone required for initial setup."
       doc: https://imss.caltech.edu/node/1408
+      
+    - name: Carnegie Mellon University
+      url: https://www.cmu.edu/
+      img: cmu.png
+      tfa: Yes
+      phone: Yes
```

Comment:
I'm looking at their docs page and I don't see mention of phone calls. Is this field accurate?

Reply:
Good catch. No I don't see phone either